config/samples/backends/hpe/3par/iscsi/backend.yaml

```diff
@@ -0,0 +1,33 @@
+# To be able to use this sample it is necessary to:
+# - Have a 3PAR backend with iSCSI support
+# - Have the 3PAR credentials in cinder-volume-3par-secrets.yaml
+# - Having iscsid and multpathd running on the host (automatically done by this sample)
+# - Build a cinder-volume container with the 3parclient cinder driver dependency, make it available in a registry accessible by OpenShift, and set its location in this file's containerImage
+#
+# There is a sample Dockerfile in the parent directory showing how the cinder-volume container in this containerImage was created.
+# The correct way to configure and run the daemons is using `MachineConfig` as shown in `iscsid.yaml` and `multipathd.yaml`, like this sample does.
+#
+# ATTENTION: After applying this OpenShift nodes will reboot, because of the `MachineConfig` changes and will take a while to recover.
+
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  cinder:
+    template:
+      cinderVolumes:
+        hpe3par-iscsi:
+          networkAttachments:
+          - storage
+          containerImage: quay.io/geguileo/podified-antelope-openstack-cinder-volume-3par:current-podified
```

Comment:
Here's another "geguileo" reference you may want to revise :-/

Reply:
I did that on purpose, because those containers actually exist and can be used by people trying those samples. ;-)

Reply:
Yes, I think it's worth putting a comment over it saying that that specific image was built using the sample `Dockerfile`